src/connectors/http.ts

```diff
@@ -94,23 +103,26 @@ export const startHttpStreamWriter: WriterConstructor<HttpWriterConfig> = (
   const requestConfig = <https.RequestOptions>new URL(config.endpoint);
 
   const push = async (item: string | Buffer): Promise<void> => {
-    await new Promise((res) => {
+    await new Promise(async (res) => {
```

Comment:
Minor detail, but perhaps rename this `res` function to, for example, `resolve` so it doesn't get confused with the `res` functions of the 2 enclosed promises (lines 123 and 124)

Reply:
That is correct, if you set that flag (default is still false)